drp/__init__.py

```diff
@@ -5,9 +5,18 @@
 from .db import db
 from .swag import swag
 
+import os
+
 
 def create_app(test_config=None):
     app = Flask(__name__)
+    app.config['UPLOAD_FOLDER'] = os.path.join(os.path.dirname(
+        app.root_path), "uploads")
+    app.config['ALLOWED_FILE_EXTENSIONS'] = {'txt', 'pdf', 'png', 'jpg',
+                                             'jpeg', 'gif', 'doc', 'docx',
+                                             'xls', 'xlsx', 'ppt', 'pptx',
+                                             'ods', 'fods', 'ods', 'fods',
+                                             'odp', 'fodp', 'md'}
```

Comment:
These configuration values should be set in config.py. For the UPLOAD_FOLDER I would also suggest an option to load the value from an environment variable for production as well as a default value as you have for development.

Reply:
Should be changed, thanks